monitoring/README.md

```diff
@@ -1,112 +1,87 @@
+<img src="images/ks-monitoring-setup.png" width="80%" height="80%" title="kubestellar-monitoring-setup">
 
 
 ### Description
-This example shows how to deploy monitoring tools (Prometheus, Grafana and node exporter) for KubeStellar components (core and edge regions) - see architecture image above. Prometheus server is deployed in the core region running the KCP server alongside the components for KCP edge. A Prometheus agent is deployed in the edge regions running the edge workload execution cluster (wec)
+This example shows how to deploy monitoring tools (Prometheus, Grafana and Pyroscope) for KubeStellar control plane components - see architecture image above. These instructions can be used to deploy monitoring tools to any KubeStellar system. If you already have the monitoring tools installed in your environment, you can skip to `step-2`.
 
-1. Create your hosts file with the list of target hosts (KCP server & Edge Workload Execution Clusters (WEC))
+1. Install the monitoring tools:
 
-```
-[kcp-server]
-192.168.56.2
+Starting from a local directory containing the git repo, do the following.
 
-[edge-wecs]
-192.160.56.10
-192.160.56.12
+```bash
+$ cd monitoring
+$ ./install-ks-monitoring.sh
 ```
 
-2. Configure the Prometheus targets endpoints:
+Optionally, check the deployment of the monitoring tools:
+
+```bash
+$kubectl -n ks-monitoring get pods
+NAME                                                  READY   STATUS    RESTARTS   AGE
+grafana-7969f6cf5c-lp4g6                              1/1     Running   0          66s
+prometheus-kube-prometheus-operator-b56d747c9-wfs2w   1/1     Running   0          42s
+prometheus-prometheus-kube-prometheus-prometheus-0    2/2     Running   0          39s
+pyroscope-0                                           1/1     Running   0          90s
+pyroscope-alloy-0                                     2/2     Running   0          90s
+```
 
-a) Prometheus Server: edit the file roles/prometheus/templates/prometheus-config.yaml.j2
+2. Configure Prometheus and Pyroscope to scrape your KubeStellar spaces:
 
+```bash
+$ ./configure-metrics-tools.sh
 ```
-global:
-  evaluation_interval: 5s
-  external_labels:
-    env: dev
-  scrape_interval: 30s
-scrape_configs:
-- job_name: mailbox-controller
-  scrape_interval: 15s
-  metrics_path: /metrics
-  static_configs:
-  - targets:
-    - '<host-ipAddress>:10203'
-
-- job_name: node-exporter
-  scrape_interval: 15s
-  metrics_path: /metrics
-  static_configs:
-  - targets:
-    - '<host-ipAddress>:9100'
-
-- job_name: kcp
-  scrape_interval: 15s
-  scheme: https
-  metrics_path: /metrics
-  tls_config:
-    insecure_skip_verify: true
-  static_configs:
-  - targets:
-    - '<host-ipAddress>:6443'
 
+Optionally, check the prometheus service monitor objects:
+
+```bash
+$kubectl -n ks-monitoring get servicemonitor -l "app.kubernetes.io/part-of!=kube-prometheus-stack"
```

Comment:
Missing space here too. And in some later commands too.

Reply:
Fixed!